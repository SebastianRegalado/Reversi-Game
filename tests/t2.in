new 6
play 1 1
play 1 4
play 3 1
play 4 3
play 4 2
play 4 1
play 3 4
play 2 1
play 4 5
play 5 2
play 5 5
play 1 3
play 5 3
play 1 2
play 4 0
play 0 1
play 3 5
play 4 4
play 3 0
play 5 1
play 0 3
play 2 5
play 2 0
play 0 2
play 2 4
play 2 0
play 5 0
play 1 0
play 0 4
play 1 5
play 0 5
play 5 4
play 0 0
