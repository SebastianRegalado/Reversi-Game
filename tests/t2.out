------
------
--BW--
--WB--
------
------
------
-B----
--BW--
--WB--
------
------
------
-B--W-
--BW--
--WB--
------
------
------
-B--W-
--BW--
-BBB--
------
------
------
-B--W-
--BW--
-BBW--
---W--
------
------
-B--W-
--BW--
-BBW--
--BW--
------
------
-B--W-
--BW--
-BWW--
-WWW--
------
------
-B--W-
--BW--
-BBBB-
-WWW--
------
------
-B--W-
-WWW--
-WWBB-
-WWW--
------
------
-B--W-
-WWW--
-WWBB-
-WWW-B
------
------
-B--W-
-WWW--
-WWBB-
-WWW-B
--W---
------
-B--W-
-WWW--
-WWBB-
-WWW-B
--W--B
------
-B-WW-
-WWW--
-WWBB-
-WWW-B
--W--B
------
-B-WW-
-WWW--
-WWBB-
-WWB-B
--WB-B
------
-BWWW-
-WWW--
-WWBB-
-WWB-B
--WB-B
------
-BWWW-
-WWW--
-WWBB-
BBBB-B
--WB-B
-W----
-WWWW-
-WWW--
-WWBB-
BBBB-B
--WB-B
-W----
-WWWW-
-WWW--
-WWBBB
BBBB-B
--WB-B
-W----
-WWWW-
-WWW--
-WWWBB
BBBBWB
--WB-B
-W----
-WWWW-
-WWW--
BBBBBB
BBBBWB
--WB-B
-W----
-WWWW-
-WWW--
BWBBBB
BWBBWB
-WWB-B
-W-B--
-WBBW-
-BWB--
BWBBBB
BWBBWB
-WWB-B
-W-B--
-WBBW-
-BWB-W
BWBBWB
BWBWWB
-WWB-B
-W-B--
-WBBW-
BBWB-W
BBBBWB
BWBWWB
-WWB-B
-WWB--
-WWBW-
BBWB-W
BBBBWB
BWBWWB
-WWB-B
-WWB--
-WWBW-
BBWBBW
BBBBWB
BWBWWB
-WWB-B
-WWB--
-WWBW-
BBWBBW
BBBBWB
BWBWWB
WWWB-B
-WWB--
BBBBW-
BBWBBW
BBBBWB
BWBWWB
WWWB-B
-WWWW-
BBBWW-
BBWBBW
BBBBWB
BWBWWB
WWWB-B
-WWWW-
BBBBBB
BBWBBB
BBBBWB
BWBWWB
WWWB-B
-WWWWW
BBBBWB
BBWWBB
BBWBWB
BWBWWB
WWWB-B
-WWWWW
BBBBWB
BBWWBB
BBBBBB
BWBBBB
WWWBBB
WWWWWW
WWBBWB
WBWWBB
WBBBBB
WWBBBB
WWWBBB
Tie!
